### ISSUE TITLE
Negative tickInterval renders the webpage unresponsive

### ISSUE THREAD

**Issue by KrishnaKumarHariprasannan** [highcharts/highcharts#12961]
#### Expected behaviour
Validate values passed as tickInterval

#### Actual behaviour
Passing negative values makes the webpage continuously increase it's memory footprint and slows down the browser

#### Live demo with steps to reproduce
https://jsfiddle.net/uhfw2gzx/

#### Product version

8.0.0

#### Affected browser(s)
Chrome, Firefox

**Comment by sebastianbochan** [comment]
Hi @KrishnaKumarHariprasannan,
Thank you for reporting about the problem. Im not sure if its a bug because the tickInterval should have the positive or undefined value.

@TorsteinHonsi should we consider it as a bug? Maybe we should only update our docs and add the information that the tickInterval has to be positive?

**Comment by KrishnaKumarHariprasannan** [comment]
Thanks for your response @sebastianbochan! I brought this up only because of the enormous load this bug puts on the webpage's compute and memory footprint. I do agree that for something as innocuous as say `yAxis.lineWidth = -10`, a note on the docs would be sufficient.

**Comment by mlucool** [comment]
Given the strain on the page (it crashes the browser sometimes), it feels like a validation should throw error rather than document that users should avoid passing this.

**Comment by raf18seb** [comment]
I agree with @mlucool, we can add some information to the docs **and** add an error handler, something like:
```js
var ...,
tickIntervalOption = isNumber(options.tickInterval) && options.tickInterval >= 0 ? options.tickInterval : undefined,
...
```
